### DISCUSSION TITLE
file was wiped - is recovery possible?

### DISCUSSION THREAD

**Discussion by MyNameIsNotListed** [gsantner/markor#1511] [Q&A] [answered]
## General information
* App version: 2.8.3
* Android version: 7.1.2 (LineageOS 14.1)
* Device: Moto G
* I understand that this issue will be closed without further notice in case I didn't describe the problem including general information, or didn't [search](https://github.com/gsantner/markor/issues?q=) for similar issues & [requests](https://github.com/gsantner/markor/discussions?discussions_q=).

### Description
I opened one .md.jenc file in edit mode, text was there, then rotated the phone to landscape orientation and got an error "file can't be decrypted...", I reopened the file and saw nothing, I thought it's a glitch and restarted the app. But the file is actually empty now. It has a size of 116 bytes, I assume it's some encryption related metadata.

When the phone is connected to Windows PC as MTP, explorer shows the previous size of that file, but if I copy it to PC it's actually 116 bytes.

---

I'm sorry if this is an off-topic but I really need an advice. Is it possible to recover the data?

I have a rough understanding of data recovery. If I understand correctly, the data is still there and I need to root the phone to access memory on a low level and use a recovery software.

But the file is not deleted, it was (probably) auto-saved empty, and it was encrypted. I don't know how to recover data in such case and if it's even possible.

**Comment by gsantner** [comment, marked as answer]
>  file was wiped. Is recovery possible?
> I'm sorry if this is an off-topic but I really need an advice. Is it possible to recover the data?

Be it encrypted or not, Markor does not make any additional copies anywhere.
So no sorry, but no recovery isn't possible.

Note: The next update comes with many file saving improvements.
